v03_pipeline/lib/paths.py

```diff
@@ -282,3 +282,10 @@ def new_variants_table_path(
         run_id,
         'new_variants.ht',
     )
+
+
+def clinvar_submission_summary_path(etag: str) -> str:
```

Comment:
Cool!  I like this, but instead of just the submission summary can we also do this for the vcf itself?  

We'd need to deal with the `ftp://` protocol that's embedded in `config.py` in a potentially janky way, but that's the real `ht` (the clinvar vcf annotated with submission summary) we want!